src/getCommands.js

```diff
@@ -8,8 +9,14 @@ const findPlugins = require('./findPlugins');
  * @param  {String}       name Name of the plugin
  * @return {Array|Object}      Plugin's commands
  */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
+const getPluginCommands = cwd => name => {
+  const nested = path.join(cwd, 'node_modules', name);
+
+  /* For rnpm installed locally with flat dependencies structure */
+  const flat = path.join(cwd, '..', name);
```

Comment:
Actually the question is whether we need that at all? Because when requiring from `rnpmRoot` for core plugins, require('name') will always resolve (either to local node_modules or to parent project ones). Same goes for app plugins.